content/markdown/03-plotting/03-05-change-over-time.rmd

```diff
@@ -0,0 +1,123 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Plotting Change Through Time
+**Author:** Cole Brookson
+**Date:** 20 July 2022
+
+
+When choosing to plot anything to do with time, the time is almost always displayed on the x-axis. This makes sense as we typically read plots as we read English language text - left to right. Thus, we will always plot time as increasing from left to right on our plots. By FAR the most common way to plot change through time is to plot it as a line plot, optionally with points added in. Less common in EEB-type data (but still useful) are box plots and bar plots.
```

Comment:
There's something that seems confused here. The section above was plotting change through time. This section maybe seems like a draft? The section above is great.

Reply:
Agree